### DISCUSSION TITLE
Update repository url: bitbucket -> gitlab

### DISCUSSION THREAD

**Discussion by tarybarna** [codemagic-ci-cd/codemagic-docs#2450] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I moved our app from Bitbucket to GitLab.

I want to keep the current configuration/envs so I need to update the repository url of the app. But the field to edit the URL is disabled.

After checking other discussions, I have created a temp app in code magic from new URL.

Old one (need repository URL update) : https://codemagic.io/app/65f028aad09b16ec9925ea55

New one (temp app, will be deleted after URL update) : https://codemagic.io/app/6608649567f9ec9423fdaa86

### Expected results

Please update the repository url.

### Actual results

Cannot do it myself.

**Comment by tarybarna** [comment]
@rafeysup93 can you please help us? i'm personally not a paying customer, so I cannot see the chat icon, but represent a team that has billing enabled, I can send you the teamId

**Comment by dtrdic** [comment, marked as answer]
Hi @tarybarna, i changed the repo url for you, please give it a try

**Reply by tarybarna** [reply]
it works, thank you
